### DISCUSSION TITLE
Openwrt路由器下阿里云盘文件夹为空

### DISCUSSION THREAD

**Discussion by renjiangsheng** [messense/aliyundrive-webdav#219] [Q&A] [answered]
你好，openwrt软路由，地址192.168.3.2, ipk安装成功，taken配置正确，局域网访问该文件夹为空。请问是挂载点或者共享还要怎么设置下吗？
![Screenshot_2021-12-25-20-37-24-58_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/60055226/147385014-e84e46ab-4fa4-449e-ba4b-705acf1aa00b.jpg)
![Screenshot_2021-12-25-20-36-58-44_846b44643ec609f507828878741e1f9a](https://user-images.githubusercontent.com/60055226/147385015-aaaf87a7-054c-4007-a4f0-c0e56656ca88.jpg)

**Comment by renjiangsheng** [comment]
版本，最新的1.1

**Comment by messense** [comment]
客户端是什么，支持 WebDAV 吗？

**Reply by renjiangsheng** [reply]
客户端是n1盒子刷的你的1.1版本阿里云插件。之前1.0的时候成功使用过，后来突然就是文件夹内容空白。今天又升级了1.1试试，还是空白，所以问下。

**Reply by renjiangsheng** [reply]
Openwrt系统

**Comment by renjiangsheng** [comment]
![Uploading Screenshot_2021-12-26-00-08-35-58_40deb401b9ffe8e1df2f1cc5ba480b12.jpg…]()

**Comment by renjiangsheng** [comment]
很奇怪。

**Comment by renjiangsheng** [comment]
@messense 继续解答下呗

**Comment by renjiangsheng** [comment]
资源管理器看为空，浏览器加8080端口正常访问。

**Comment by renjiangsheng** [comment]
资源管理器看为空，浏览器加8080端口正常访问。是openwrt哪里设置问题吗？
github说的太粗放了，能否梳理下步骤？

**Comment by messense** [comment, marked as answer]
你没有回答清楚上面的问题，你所谓的资源管理器是什么，支持 WebDAV 吗？建议换个明确支持的 WebDAV 的客户端 App

**Reply by renjiangsheng** [reply]
前面说的是Windows系统下的文件管理器。
本回复截图是安卓下分别通过浏览器访问和通过安卓平台es文件管理器的局域网功能。
浏览器访问正常(地址加了8080端口)，说明token正确。
文件管理器空白，求解？
![Uploading Screenshot_2022-01-03-12-24-22-18_846b44643ec609f507828878741e1f9a.jpg…]()

**Reply by renjiangsheng** [reply]
![Screenshot_2022-01-03-12-24-22-18_846b44643ec609f507828878741e1f9a](https://user-images.githubusercontent.com/60055226/147899787-42d71bff-4c73-4c11-86ba-2ebec83cc112.jpg)
![Screenshot_2022-01-03-12-24-05-76_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/60055226/147899791-14dcba90-7050-425c-b5f3-a2ff8860f06c.jpg)
如图。

**Reply by renjiangsheng** [reply]
或者确认下，openwrt系统配置成功后，局域网里的windows电脑或者安卓手机该如何访问阿里云呢？
是要在openwrt分别新建挂载点和共享吗？

**Reply by messense** [reply]
https://github.com/messense/aliyundrive-webdav/discussions/197

**Reply by renjiangsheng** [reply]
肯定支持的

**Reply by orsonleu** [reply]
> 肯定支持的

你好 请问你的问题解决了吗 我也出现了相同的问题

**Reply by tbx8564** [reply]
> > 肯定支持的
> 
> 
> 
> 你好 请问你的问题解决了吗 我也出现了相同的问题

你的解决了没？我也是这样

**Comment by bananahotel** [comment]
请问op解决了吗，我最近才用，也有这个问题
